### PR TITLE
Updating content per recomendations

### PR DESCRIPTION
**Pull request by line47**
general content updates made based on recommendations. Updating to the new application name of "Toaster"

### CODE REVIEW

File: app/views/layouts/application.slim [hunk 1]
```diff
@@ -2,7 +2,7 @@ doctype html
 html lang='en'
   head
     title
-      = content_for?(:title) ? yield(:title) : t('description')
+      = content_for(:title) ? "Toaster: " + yield(:title) : t('title')
```
Comment (jessieay) [inline app/views/layouts/application.slim]: I thought we were going to put this in a locale string?

File: app/views/messages/_form.slim [hunk 2]
```diff
@@ -5,7 +5,8 @@
   label.no-top-margin for="message_body"
     strong = f.label(:message_label_text, class: 'no-top-margin')
   span.usa-form-hint = t('helpers.label.message.dont_sent_hint') + " "
-  = link_to 'content that would violate federal rules', about_path
+  = link_to 'content that could violate federal rules.', about_path
```
Comment (jessieay) [inline app/views/messages/_form.slim]: make this localized?

File: app/views/pages/about.html.slim [hunk 3]
```diff
@@ -16,13 +18,14 @@
   p = t('layout.info.about.p3')
   p = t('layout.info.about.p4')
   h2 = t('layout.info.about.p6')
-  ul
-    li = t('layout.info.about.p7')
-    li = t('layout.info.about.p8')
-    li = t('layout.info.about.p9')
-    li = t('layout.info.about.p10')
+  ul 
+    - t('layout.info.about.do_list').each do |item|
```
Comment (jessieay) [inline app/views/pages/about.html.slim]: cool! I've not used lists in I18n before, but I think this is a good time to do that

File: config/locales/en.yml [hunk 4]
```diff
@@ -21,31 +21,34 @@ en:
     info:
       nav:
         about: "How this works"
-        privacy: "Privacy Policy"
-        github: "Github"
+        privacy: "Privacy policy"
+        github: "GitHub"
```
Comment (jessieay) [inline config/locales/en.yml]: I feel like you changed this in another PR -- did you sync with the dev branch?

File: app/views/layouts/application.slim (hunk 1 above)
Comment (line47) [inline app/views/layouts/application.slim]: Oh yeah! Totally forgot about that. I'll update.

File: app/views/messages/_form.slim (hunk 2 above)
Comment (line47) [inline app/views/messages/_form.slim]: I'm unsure how I would write the syntax here as this is a sentence with part of it being text and the other page being a link. 

screen shot 2017-03-07 at 4 18 54 pm

File: config/locales/en.yml (hunk 4 above)
Comment (line47) [inline config/locales/en.yml]: That was an older PR https://github.com/fugacious/fugacious/pull/118 which was a bit scattered and still had work to be done so I closed it.

File: app/views/messages/_form.slim (hunk 2 above)
Comment (jessieay) [inline app/views/messages/_form.slim]: why not something like 

`= link_to I18n.t('helpers.label.message.dont_sent_hint_2`), about_path`

File: app/views/layouts/application.slim (hunk 1 above)
Comment (jessieay) [inline app/views/layouts/application.slim]: let me know when you've done this

**Comment by jessieay** [comment]
It seems like this contains some but not all of https://github.com/fugacious/fugacious/pull/118/files -- is the point of this PR to update naming to "Toaster? Would be helpful to clarify in the PR description beyond the fact that this is general "content updates"

**Comment by line47** [comment]
@jessieay Yeah, that's exactly right. I left #118 as it is because it got into naming that I was unsure about changing. I have a few conflicts on this branch now that I'm going to look to resolve today so we can hopefully merge.

File: app/views/layouts/application.slim (hunk 1 above)
Comment (line47) [inline app/views/layouts/application.slim]: Got it 👍

File: app/views/messages/new.slim [hunk 5]
```diff
@@ -1,5 +1,5 @@
-- title t('site_title')
-
+-title t('description') 
+
```
Comment (jessieay) [inline app/views/messages/new.slim]: can you remove this whitespace character? if you don't already, I highly recommend turning on "show whitespace" in your text editor :)

File: app/views/layouts/application.slim [hunk 6]
```diff
@@ -2,7 +2,7 @@ doctype html
 html lang='en'
   head
     title
-      = content_for?(:title) ? yield(:title) : t('description')
+      = content_for(:title) ? t('site_title') + " - " + yield(:title) : t('title')
```
Comment (jessieay) [inline app/views/layouts/application.slim]: 👍

File: app/views/messages/new.slim (hunk 5 above)
Comment (line47) [inline app/views/messages/new.slim]: now showing whitespace 👍 and updated.

**Pull request merged**